docs/template/styles/docfx.js

```diff
@@ -721,20 +721,20 @@ $(function () {
     function initFooter() {
       if (needFooter()) {
         shiftUpBottomCss();
-        // $("footer").show();
+        $("footer").show();
       } else {
         resetBottomCss();
-        // $("footer").hide();
+        $("footer").hide();
       }
     }
 
     function showFooterCore() {
       if (needFooter()) {
         shiftUpBottomCss();
-        // $("footer").fadeIn();
+        $("footer").fadeIn();
       } else {
         resetBottomCss();
-        // $("footer").fadeOut();
```

Comment:
@eaba this was disabled so the footer would stay persistent at the bottom of the site. Why did you add it back?